tests/pylint/runpylint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python
```

Comment:
Contradicts what code? What environment are you running this in? Anything that changes python to python3 is going to cause no end of problems. Use python3 for python3 code.